src/envoy/http/service_control/http_call.cc

```diff
@@ -72,32 +72,40 @@ class HttpCallImpl : public HttpCall,
   // HTTP async receive methods
   void onSuccess(Http::ResponseMessagePtr&& response) override {
     ENVOY_LOG(trace, "{}", __func__);
-    const uint64_t status_code =
```

Comment:
Diff here is messy. All I did was move this code inside a try block to handle missing `:status` header.